src/main/java/uk/gov/beis/els/service/AnalyticsService.java

```diff
@@ -85,4 +84,24 @@ public void sendGoogleAnalyticsEvent(String jsClientId, GoogleAnalyticsEventCate
 
   }
 
+  // Handle secrets set via GovUk PaaS. Can be removed once migrated to BEIS AWS platform
+  // See https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES and
+  // https://docs.cloudfoundry.org/devguide/services/user-provided.html
+  private String parseGaApiSecret() {
+    if (System.getenv("VCAP_SERVICES") == null) {
+      // Allow running locally when PaaS vars don't exist
+      LOGGER.warn("No Gov PaaS VCAP_SERVICES env var found. Unable to extract GA api secret.");
+      return "not-set";
```

Comment:
Could we perhaps have a `local_dev_analytics_api_secret` property to fall back to so that the API key can be set in the properties file for local development?

Reply:
I've added this in. Not in the nicest way but it will all be changing soon once we move off PaaS and we can just read secrets from the env directly, rather than this weird CF method.